.github/workflows/automerge-pyup.yml

```diff
@@ -0,0 +1,37 @@
+name: "pyup Automerge"
+
+on:
+  pull_request:
+
+jobs:
+  automerge:
+    runs-on: ubuntu-latest
+    
+    if: github.actor == 'pyup-bot'
+    steps:
+      - name: 'Wait for status checks'
+        id: waitforstatuschecks
+        uses: "WyriHaximus/github-action-wait-for-status@master"
+        with:
+          ignoreActions: automerge,WIP
+          checkInterval: 300
+        env:
+          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+
+      - name: 'automerge'
+        uses: actions/github-script@0.2.0
+        if: steps.waitforstatuschecks.outputs.status == 'success'
+        with:
+          script: |
+            github.pullRequests.createReview({
+              owner: context.payload.repository.owner.login,
+              repo: context.payload.repository.name,
+              pull_number: context.payload.pull_request.number,
+              event: 'APPROVE'
+            })
+            github.pullRequests.merge({
+              owner: context.payload.repository.owner.login,
+              repo: context.payload.repository.name,
+              pull_number: context.payload.pull_request.number
+            })
+          github-token: ${{github.token}}
```

Comment:
We can definitely use that instead. Just thought that I would mess up the configuration had I used package instead since I was basically only testing that deleted portion, which worked fine for me. Plus, GitHub scripts allows you to write some more stuff in case you wanted any more custom code, so I just left this as is. If you mind, I can commit the suggestion.

Reply:
> GitHub scripts allows you to write some more stuff in case you wanted any more custom code with the GitHub API

Yea, that could be useful if we had a need for it, but I don't think it's the case here... Just thinking that I'd rather use something with a higher level of abstraction, whenever possible.

> actually I'd have to rewrite some of it so that the package only merges with the label "update" by pyup; otherwise it looks for auto merge. 

Do you mean that you want to check for the `update` label as well `puyp` user? Not sure it's necessary, checking the author should be enough IMO.

Reply:
No no, uh I believe the action will only merge those with the label "auto merge" unless you write for one of its env variables MERGE_LABELS to be "update"

Reply:
Aaaah got it, missed that detail in their readme. Good catch 👍

Reply:
No problem, incorporated in latest commit and tested again in that test repo.